### ISSUE TITLE
Setting default builder (newbie) question

### ISSUE THREAD

**Issue by joep** [trailblazer/formular#46]
Hi,

I just starting formular for a Sinatra app. Using the suggested

`helpers Formular::Helper`

works fine to get the form helper into my templates.

However, using the suggested

`Formular::Helper.builder = :bootstrap3`

to set the defaut builder doesn't work for me.

```
puts   Formular::Helper.builder.inspect
Formular::Helper.builder = :bootstrap3
puts   Formular::Helper.builder.inspect
```
Outputs Formular::Builders::Basic for both.

Thanks!

**Comment by emaglio** [comment]
Are you using the master branch? We should have fixed that with one of the latest commit.

**Comment by fran-worley** [comment]
I'll bump the version :)

**Comment by joep** [comment]
Switching to the master branch did fix the issue.

Thanks!

**Comment by fran-worley** [comment]
@joep I've just released v0.2.2 which includes the helper fixes.
